src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1742,6 +1743,21 @@
                         </UserData>
                     </template>
 
+                    <template tid="ContentionStart_V2">
+                        <data name="ContentionFlags" inType="win:UInt8" map="ContentionFlagsMap" />
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="LockObjectID" inType="win:Pointer" />
```

Comment:
Recommend calling this `LockID`, and then match with the same name in the `AwareLockCreated` event.